source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -1,53 +1,63 @@
 Viewing Diagnostics
 ===================
 
-The first diagnostic to look at is the r-squared - it should be somewhere north of ~.9. If it is significantly lower than this, there is likely a problem with your identification data.
+Goodness-of-Fit Metrics
+-----------------------
+
+There are three numerical accuracy metrics that are computed with this tool: acceleration r-squared, simulated velocity r-squared, and standard deviation of the simulated velocity error.
```

Comment:
This line now mentions standard deviation but below it still talks about RMSE.  I am pretty sure those aren't the same thing.  Which is it doing?

Reply:
They are indeed the same thing.  The RMSE is the standard deviation of the error...